coreMQTT/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e
+
+COREMQTT_VERSION="2.3.0"
+COREMQTT="coreMQTT-${COREMQTT_VERSION}"
+PKG_URL="https://github.com/FreeRTOS/coreMQTT/archive/refs/tags/"
+
+b_log "Building coreMQTT"
```

Comment:
Not needed. phoenix-rtos-ports already prints this before executing the port's build.sh

Reply:
Done